content/zh/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -230,7 +223,7 @@ curl -X DELETE localhost:8080/apis/apps/v1/namespaces/default/replicasets/my-rep
 Here's an example that deletes dependents in foreground:
 -->
 
-下面是一个在前台删除附属对象的例子：
+下面是一个在 `Foreground` 中删除附属对象的示例：
```

Comment:
```suggestion
下面是一个在前台删除附属对象的示例：
```

Reply:
see https://github.com/kubernetes/website/pull/18865/files#diff-563c4c44553d361a4d472a93dd458b7bR205

I think it ’s better not to translate here